configs/components/rubygem-puppet.rb

```diff
@@ -1,6 +1,6 @@
 component 'rubygem-puppet' do |pkg, settings, platform|
-  pkg.version "6.19.1"
-  pkg.md5sum '8d68e677e6ba88e46eb00699dcd4930b'
+  pkg.version "7.0.0"
```

Comment:
I think we'll actually want to update to latest, 7.3.0. Does Bootsnap depend on this? It might be good to open a separate PR since I think Skeletor will have to compile Bolt with a different version of Puppet than Puppet 7.

Reply:
I should have documented it in the commit message, my bad, but we need puppet 7.X because it removed the win32-dir gem. Bootsnap uses a hash lookup for finding cached file paths, and the gem folder name `win32-dir` does not match the gem require statement of `win32/dir`.

I'm fine with bumping to pupet 7.X in another PR before merging this.